.github/workflows/acc-tests.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             GRAFANA_AUTH=cloud-instance-tests:auth
             GRAFANA_ONCALL_ACCESS_TOKEN=cloud-instance-tests:oncall-token
             GRAFANA_SM_ACCESS_TOKEN=cloud-instance-tests:sm-token
+            GRAFANA_K6_ACCESS_TOKEN=cloud-instance-tests:k6-token
```

Comment:
Do you know where/how should I define/create this? Thanks! 🙏🏻 

cc/ @spinillos @Duologic

Reply:
Also, note that k6 resources rely on `stack_id`, is that set for "cloud instance tests"? 🤔

Reply:
You can create a new secret in Vault at `ci/repo/grafana/terraform-provider-grafana/<secret name>` then referrence it here as `GRAFANA_K6_ACCESS_TOKEN=<secret name>:k6-token`.

I'm not sure to which 'stack' is referred with `stack_id`, in the docs on this PR I see that k6 also has a 'stack' concept... either way, if you know the id upfront you can also put it in the same secret and read it here.

Reply:
How I'm supposed to know in what stack do I need to create the k6 token, if I can access the secret in use?
I guess, here, I'd need to know `GRAFANA_URL` and likely `GRAFANA_AUTH` (to log in and generate the token?).

Then, I hope to be able to create a new secret at `ci/repo/grafana/terraform-provider-grafana/...`.

Reply:
It worked, thanks @Duologic for your assistance! 💟 🙇🏻